### DISCUSSION TITLE
OData response for Dynamic Model with EF

### DISCUSSION THREAD

**Discussion by rbalagangadharan** [OData/AspNetCoreOData#1216] [Q&A]
Hey all,

I am working on dynamic EDM with EF, where I am generating the EDM at run time and returning the data from DB using EF. Everything works fine, but I am not getting the response in OData format. see the actual response below, where the odata.context, odata.nextlink is not getting generated. How to achieve this?

Actual response.
```
[
    {
        "Name": "ABC"
    }
]
```

expected response 

```
{
    "@odata.context": "https://localhost:7181/odata/Test/$metadata",
    "value": [
        {
            "Name": "ABC"
        }
    ]
}
```

**Comment by julealgon** [comment]
You'll need to share more details about your implementation otherwise it is nearly impossible to know what is going on in your side @rbalagangadharan .

It is likely the case that your routes are not matching your EDM but we need to see your code to understand why.
